src/configurator/v1.ml

```diff
@@ -152,6 +157,60 @@ module Process = struct
   let command_line prog args =
     String.concat ~sep:" " (List.map (prog :: args) ~f:quote_if_needed)
 
+  let run_process t ?dir ?env prog args =
+    let args_str = String.concat ~sep:" " args in
```

Comment:
Why not use `command_line` here? At least, we will get a valid command in case of error.